demisto_sdk/tests/integration_tests/xsoar_linter_integration_test.py

```diff
@@ -48,7 +48,6 @@
         True,
         1,
         [
-            "Print is found, Please remove all prints from the code.",
```

Comment:
why? this lints content code

Reply:
https://github.com/demisto/demisto-sdk/pull/2917/files#diff-3602faaca6fa8fe18eb2820af2c85f6e2463d5d695040386db3221ad99917746